src/status/myDomainResolver.ts

```diff
@@ -65,7 +65,7 @@ export class MyDomainResolver extends AsyncOptionalCreatable<MyDomainResolver.Op
 
         try {
           self.logger.debug(`Attempting to resolve url: ${host}`);
-          if (host && host.includes('.internal.salesforce.com')) {
+          if (host?.includes('.internal.salesforce.com') || host?.includes('localhost')) {
```

Comment:
This is scattering knowledge about localhost between here and isInternalUrl.

Can this be consolidated?

Reply:
yes...but I'm not sure they're exactly the same.  This is more isLocal...link things we don't want to dns check.

you can have internal steam/vpod/etc that still need to verify DNS resolution, correct?

Reply:
proposed: isLocal, which is both exported for use by myDomainResolver and plugins AND used by isInternal

Reply:
@mshanemc internal, non local need to be checked, but having an isLocal check for just localhost seems reasonable to me.

Reply:
updated